server.babel.js

```diff
@@ -1,8 +1,9 @@
 //  enable runtime transpilation to use ES6/7 in node
 
 var fs = require('fs');
+var path = require('path');
 
-var babelrc = fs.readFileSync('./.babelrc');
+var babelrc = fs.readFileSync(path.resolve(__dirname, 'node_modules/universal-redux/.babelrc'));
```

Comment:
I don't know. It doesn't feel right to me. Could not be posible do it in another way?

What's the problem with having `.babelrc` here, in this project. And another `.babelrc` in `universal-redux` module?

Sorry, I just want to understand it